charts/microgateway/README.md.gotmpl

```diff
@@ -225,18 +227,23 @@ By default, the Airlock Microgateway is configured with the [Simple DSL configur
 In case that the [Simple DSL configuration](#simple-dsl-configuration) does not suite, the advanced configuration options might help. The following use cases might require this kind of configuration:
 
 **_Use Case 1)_**
-* Virtual Host 1: abc.com -> Mapping 1: / -> Backend Service 1
-* Virtual Host 2: xyz.com -> Mapping 2: / -> Backend Service 1
+| Virtual Host | Mapping | Backend Service |
+|--|--|--|
+| VH1, FQDN: virtinc.com | M1, Path: / | BE1 |
```

Comment:
I would paraphrase this so that a bracket is used instead of a comma. 
VH1, FQDN: virtinc.com -->
virtinc.com (VH1) or VH1 (virtinc.com)

M1, Path: / -->
M1 (entryPath: /)